src/TestEnvironment.Docker.Containers.Oracle/OracleContainerWaiter.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Oracle.ManagedDataAccess.Client;
+
+namespace TestEnvironment.Docker.Containers.Oracle
+{
+    public class OracleContainerWaiter : BaseContainerWaiter<OracleContainer>
+    {
+        public OracleContainerWaiter(ILogger logger)
+            : base(logger)
+        {
+        }
+
+        protected override async Task<bool> PerformCheck(OracleContainer container, CancellationToken cancellationToken)
+        {
+            using var connection = new OracleConnection(container.GetConnectionString());
+            using var command = new OracleCommand("SELECT * FROM V$VERSION", connection);
+
+            await connection.OpenAsync(cancellationToken);
+            await command.ExecuteNonQueryAsync(cancellationToken);
```

Comment:
Tests are failing with error: 
```
test-env-my-oracle check failed with exception ORA-00604: error occurred at recursive SQL level 1
      ORA-01882: timezone region not found
Oracle.ManagedDataAccess.Client.OracleException (0x80004005): ORA-00604: error occurred at recursive SQL level 1
ORA-01882: timezone region not found
```

Probably you need to specify additional parameters to default connection string or configure IsRetryable property in the similar way as in https://github.com/Deffiss/testenvironment-docker/blob/master/src/TestEnvironment.Docker.Containers.MariaDB/MariaDBContainerWaiter.cs#L29

Reply:
True, this is how timezome might be set on connection level
https://stackoverflow.com/questions/53510011/asp-net-core-docker-container-using-oracle-managed-driver-core-throws-ora-00604

Reply:
Thanks, I tried to add the timezone in the env vars, let's see if that's enough

Reply:
It looks like the AppVeyoir build timed out :(

Reply:
There are still `ORA-01882: timezone region not found` errors in appveyor, I suspect there is no `UTC` timezone inside Docker image. I'll try to troubleshoot locally.

Reply:
Yes, I should have check available timezone names before. I don't have the issue on my computer, maybe the local timezone is passed (Docker with WSL 2)

Reply:
I played a bit with your fork. It looks like the problem is not on container side but on connection. I do not have permissions to push potential fix, so could you try please:
`
var info = connection.GetSessionInfo();
info.TimeZone = "UTC";
connection.SetSessionInfo(info);
`
In two places:
1. `OracleContainerWaiter`, line 22
2. `PrintOracleVersion`, line 372

Also, you can probably remove TZ set in dictionary on line 26 in `DockerEnvironmentBuilderExtensions`

Reply:
Sure

Reply:
Still no luck ... Maybe with a more "common" timezone like "Europe/Paris" ? I checked, UTC exists in the Oracle timezone table though.

Reply:
Yes, very strange. Locally it works just fine.

Reply:
I did some research and found that in `HealthCheck` project Oracle tests are skipped for some reasons on AppVeyor:
https://github.com/Xabaril/AspNetCore.Diagnostics.HealthChecks/blob/master/test/FunctionalTests/HealthChecks.Oracle/OracleHealthCheckTests.cs

Reply:
My current feeling that in AppVeyor there is a discrepancy between timezones inside container and on builder machine, we need to find a way to identify timezone that exists in two places.